### PR TITLE
Updates to distributed databases article

### PR DESCRIPTION
**Pull request by sklarsa**

### CODE REVIEW

File: blog/2023-03-03-how-reliable-is-your-distributed-database.mdx [hunk 1]
```diff
@@ -15,65 +15,45 @@ tags: [replication, comparison, engineering]
 slug: how-reliable-your-distributed-database
 ---
 
-A brief analysis of what can go wrong with databases and how different
-time-series databases can use replication to solve common write failover
-scenarios.
+A introduction to how replication works in distributed databases,
+and how common write failover scenarios are handled by real time-series databases.
```
Comment (tris0laris) [inline blog/2023-03-03-how-reliable-is-your-distributed-database.mdx:19]: real time-series ?

File: blog/2023-03-03-how-reliable-is-your-distributed-database.mdx [hunk 2]
```diff
@@ -15,65 +15,45 @@ tags: [replication, comparison, engineering]
 slug: how-reliable-your-distributed-database
```
Comment (tris0laris) [inline blog/2023-03-03-how-reliable-is-your-distributed-database.mdx:15]: tags, i would add: time-series, database, distributed, high availability

Comment (tris0laris) [inline blog/2023-03-03-how-reliable-is-your-distributed-database.mdx:15]: inner-workings-distributed-databases

File: blog/2023-03-03-how-reliable-is-your-distributed-database.mdx (hunk 1 above)
Comment (sklarsa) [reply blog/2023-03-03-how-reliable-is-your-distributed-database.mdx:19]: yeah I guess I don't need the "real"..

File: blog/2023-03-03-how-reliable-is-your-distributed-database.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: How Reliable is Your Distributed Database?
+title: The Inner Workings of Distributed Databases
```
Comment (tris0laris) [inline blog/2023-03-03-how-reliable-is-your-distributed-database.mdx:2]: for keywords, i would remove "comparison"

**Review by tris0laris — APPROVED** [review]

**Pull request merged**